src/components/common/Extensions/Navigation/Navigation.tsx

```diff
@@ -7,15 +8,18 @@ const displayName = 'common.Extensions.Navigation';
 
 const Navigation: FC = () => {
   const isMobile = useMobile();
+  const navigationItems = useSideNavigation();
+  const { pathname } = useLocation();
+
+  const currentPage = navigationItems.filter((element) => element.linkTo === pathname);
```

Comment:
maybe getCurrentPage will be better name

Reply:
It's not a function, I've changed it to currentPathname